src/run-tests.xml

```diff
@@ -0,0 +1,407 @@
+  if bustedState.runTestsRun then
+    local tablex = require 'pl.tablex'
+    local root = busted.context.get()
+    local children = tablex.copy(busted.context.children(root))
+    local oldctx = busted.context.get()
+    busted.context.clear()
+    local ctx = busted.context.get()
+    for k, v in pairs(oldctx) do
+      ctx[k] = v
+    end
+    for _, child in ipairs(children) do
+      for descriptor, _ in pairs(busted.executors) do
+        child[descriptor] = nil
+      end
+      busted.context.attach(child)
+    end
+    root = busted.context.get()
+    busted.safe_publish('suite', { 'suite', 'reset' }, root, 1, 1)
+  end
+  bustedState.runTestsRun = true
+
+  -- redirect io.write() to main screen temporarily
+  -- this could have side-effects if other threads or the test cases use
+  -- io.write() (which normally writes to the command line) before we 
+  -- reset it once the tests are done.
+  oldiowrite = io.write
+  io.write = function (...)
+    echo(...) 
+  end
+
+  -- this is the actual busted call that runs the tests, which were
+  --   loaded in bustedState.setup.
+  bustedState.execute(1, {})
+
+  io.write = oldiowrite
+end -- bustedState.runTests</script>
+      <eventHandlerList />
+    </Script>
+    <Script isActive="no" isFolder="no">
+      <name>README</name>
+      <packageName></packageName>
+      <script>-- This is the set of mudlet scripts to allow running Busted tests from
+-- within the hosted lua environment of Mudlet.
+-- Busted is "a unit testing framework with a focus on being easy to use."
+-- It is written by olivine Labs, and documentation can be found on
+-- their website: https://olivinelabs.com/busted/
+-- 
+-- Using busted within Mudlet's hosted environment allows you to test
+-- lua functions defined within the environment, both user created functions
+-- (You can test your scripts!) and builtin functions.
+--
+-- So how do we get this to work?
+-- Second, you need to install Busted on your system.
+-- Huh? Second? Yes; since Busted is a luarocks package:
+-- First, you need to install luarocks on your system.
+--
+-- For instructions on installing luarocks:
+-- On unix see:  https://luarocks.org/#quick-start
+-- On Windows see: https://github.com/luarocks/luarocks/wiki/Installation-instructions-for-Windows
+--
+-- You can test that you have luarocks installed by typing 
+-- 'luarocks --version' (no quotes) on a command prompt then hit enter.
+--
+-- Once you have luarocks installed, installing busted is easy.
+-- From a command prompt, type 'luarocks install busted' (no quotes), then
+-- hit enter.
+--
+-- Now, once THAT's all done, you can write and run tests.
+--
+-- The writing of Busted tests is covered in the documentation referenced
+-- above, but to get you started, here's an example
+--
+--[[
+-- ** Start of file **
+describe("should always pass", function()
+    it("is the simplest test I could think of.", function()
+        assert.truthy("Yup.")
```

Comment:
Could you add a 1 == 1 here or something so it's more obvious? Testing that a Lua string isn't nil which equals true is kind of advanced knowledge.